org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/text/folding/DefaultJavaFoldingStructureProvider.java

```diff
@@ -1011,8 +1027,13 @@ public void propertyChange(PropertyChangeEvent event) {
 	private boolean fCollapseInnerTypes= true;
 	private boolean fCollapseMembers= false;
 	private boolean fCollapseHeaderComments= true;
+	private boolean fCollapseCustomRegions= false;
 	private boolean fNewFolding;
 
+	private boolean fCustomFoldingRegionsEnabled= true;
```

Comment:
Curious why this isn't false by default, but perhaps it will make sense later on.

Reply:
I just followed using the defaults in the method above. This shouldn't make much of a difference since it should be set by `initializePreferences()` but I guess it's better to be `false` since `fCustomFoldingRegionBegin` and `fCustomFoldingRegionEnd` are not initialized (fixed).